synapse/config/experimental.py

```diff
@@ -46,3 +46,11 @@ def read_config(self, config: JsonDict, **kwargs):
 
         # MSC3266 (room summary api)
         self.msc3266_enabled: bool = experimental.get("msc3266_enabled", False)
+
+        # MSC2409 (this setting only relates to optionally sending to-device messages).
```

Comment:
it is ... frustrating... that it doesn't apply to other things covered by MSC2409. still, not relevant here.

Reply:
no, I think that will be more confusing. I'm just grumbling.